packages/server/graphql/queries/helpers/fetchJiraServerProjects.ts

```diff
@@ -0,0 +1,36 @@
+import {DataLoaderWorker} from './../../graphql'
+import JiraServerRestManager from '../../../integrations/jiraServer/JiraServerRestManager'
+
+const fetchJiraServerProjects = async (
+  teamId: string,
+  userId: string,
+  dataLoader: DataLoaderWorker
+) => {
+  const token = await dataLoader
+    .get('teamMemberIntegrationAuths')
+    .load({service: 'jiraServer', teamId, userId})
+  if (!token) return []
+  const provider = await dataLoader.get('integrationProviders').loadNonNull(token.providerId)
+
+  const manager = new JiraServerRestManager(
+    provider.serverBaseUrl!,
+    provider.consumerKey!,
+    provider.consumerSecret!,
+    token.accessToken!,
+    token.accessTokenSecret!
```

Comment:
+1 it looks like these five fields are non-nullable so we can drop the non-null assertions

Reply:
@nickoferrall as I see, they could be nullable, removing `!` cause typescript error

Reply:
Yeah, you're right - it looks like it's nullable: https://github.com/ParabolInc/parabol/blob/fbb67ac58ed3c64bc8ff7c122b6facb64bb08131/packages/server/postgres/queries/generated/getTeamMemberIntegrationAuthQuery.ts

My editor must have been playing games